hooks/scripts/check-print-keyword.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+## Check for print keyword used for logging in the repository.
+
+count=`grep -r 'print\s*(' . | wc -l`
+
+if [[ ! $count -ge 1 ]]; then
```

Comment:
@KritikaA06  We should fail if the count is greater than or equal to 1. Seems to be a typo.

```suggestion
if [[ $count -ge 1 ]]; then
```

Reply:
yes typo. thanks!